examples/thermostat/silabs/src/AppTask.cpp

```diff
@@ -173,3 +183,67 @@ void AppTask::ButtonEventHandler(uint8_t button, uint8_t btnAction)
         sAppTask.PostEvent(&aEvent);
     }
 }
+
+#ifdef DIC_ENABLE
+void AppTask::DIC_AttrubiteHandler(AttributeId attributeId)
```

Comment:
Why is it in Apptask? Why not your own DIC project files? It could probably be merged with the tempMgr attribute change handling also no?